src/Caching/Storages/SQLiteStorage.php

```diff
@@ -154,6 +153,16 @@ public function clean(array $conditions): void
 			}
 
 			$this->pdo->prepare($sql)->execute($args);
+		} elseif (!empty($conditions[Cache::NAMESPACE])) {
+			if (is_array($conditions[Cache::NAMESPACE])) {
+				$namespaces = $conditions[Cache::NAMESPACE];
+			} else {
+				$namespaces = [$conditions[Cache::NAMESPACE]];
+			}
+
+			foreach ($namespaces as $namespace) {
+				$this->pdo->prepare("DELETE FROM cache WHERE key LIKE %/_$namespace/%")->execute();
```

Comment:
This is sql injection, it should be

`$this->pdo->prepare('DELETE FROM cache WHERE key LIKE ?)')->execute([$namespace . Cache::NAMESPACE_SEPARATOR])`